docs/SYNC_MODE.md

```diff
@@ -32,11 +32,68 @@ Taps can support two replication methods, and should decide which to use by chec
 ## Stream/Field Selection
 Taps should allow users to choose which streams and fields to replicate. The following metadata should be checked to decide whether a stream/field should be replicated:
 
-| Metadata Keyword | Description  | 
+| Metadata Keyword | Description  |
 | ----------------- | ------- |
 | `inclusion` | Only applies to fields.  If this is set to `automatic`, the field should be replicated.  If this is set to `unsupported`, the field should not be replicated.  Can be written by a tap during discovery |
 | `selected` | If this is set to `True`, the stream (empty breadcrumb), or field should be replicated.  If `False`, the stream or field should be omitted.  This metadata is written by services outside the tap. |
+| `available` | The field may or may not be `selected`. |
 
+Streams shouldn’t be selected by default. We want them to have an `inclusion: available` property. Each field needs to have its own metadata object that labels the field as `inclusion: automatic` or `inclusion: available`.
+
+## Suggested Pattern for implementing stream selection
+You can get all the proper inclusion metadata during discovery by calling `get_standard_metadata()` as is done here:
+```
+ mdata = metadata.get_standard_metadata(
+            schema=<schema>,
+            key_properties=<key_properties>,
+            valid_replication_keys=<replication_keys>,
+            replication_method=<replication_method>,
+        )
+```
+The `mdata` object can then be passed to the catalog object.
```

Comment:
May be good to elaborate a bit on the format of the return value, like "The returned `mdata` object is in list format and can be passed directly to the `Catalog` object constructor."